assembl/static2/flow/README

```diff
@@ -0,0 +1,11 @@
+The graphql_types.flow.js was generated with the following commands:
+
+    assembl-pshell local.ini
+    from assembl.graphql.schema import generate_schema_json; generate_schema_json()
+    ctr+d
+    cd assembl/static2
+    yarn add apollo-codegen
+    ./node_modules/.bin/apollo-codegen generate js/app/graphql/*.graphql js/app/graphql/mutations/*.graphql --schema /tmp/schema.json --target flow --output flow/graphql_types.flow.js
+    git checkout package.json yarn.lock
+    rm -rf node_modules/
+    yarn
```

Comment:
I install apollo-codegen only to generate the file, then I restart fresh without it. I have lots of flow errors if I keep it around, probably a flow version mismatch...